app/views/layouts/application.html.haml

```diff
@@ -34,12 +34,11 @@
               = render 'application/breadcrumbs'
             = content_for?(:content) ? yield(:content) : yield
         %aside.wrappable__content.promotions
-          %section.promotion{ style: 'margin-top: 20px;'}
-            - if current_site.ad_client.present? && current_site.ad_slot.present?
-              %section.promotion
+          %section.promotion
+            %div.ad
```

Comment:
I noticed this tag for not only adsense but also dfp
Hmm, so we should name it `.promotion--ad` or `.promotion--banner`